velodyne_driver/src/driver/velodyne_node.cc

```diff
@@ -35,22 +35,26 @@
  *  ROS driver node for the Velodyne 3D LIDARs.
  */
 
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 #include "velodyne_driver/driver.h"
 
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, "velodyne_node");
-  ros::NodeHandle node;
-  ros::NodeHandle private_nh("~");
+  // Force flush of the stdout buffer.
+  setvbuf(stdout, NULL, _IONBF, BUFSIZ);
```

Comment:
Is there a reason this is necessary?

Reply:
Hm probably not, it's just in every ROS2 sample, so I just thought it was something that was best practice if you wanted to log to stdout in C++ ? But I can remove it,

Reply:
I haven't found anything specifically stating that this is necessary. I guess it doesn't really hurt to leave it.